core/storage/suggestion/gae_models.py

```diff
@@ -821,7 +825,18 @@ def get_in_review_question_suggestions_by_offset(
                     break
                 for suggestion_model in suggestion_models:
                     offset += 1
-                    if suggestion_model.author_id != user_id:
+
+                    included_in_topics = False
+                    if topics is not None:
+                        for topic in topics:
+                            # ERROR in line below: 'Topic' object has no attribute 'question_ids'
```

Comment:
Hi @sagangwee, I need to find the relationship here between Suggestion and Topic (probably, there is a Skill in the middle, but not sure how many other models)

Any suggestion on how to easily find relationships between models? Any Oppia entity-relationship diagram or models visualization available?

Reply:
This page might be helpful: https://oppia-user-guide.readthedocs.io/en/latest/keyconcepts.html. But basically:

- a Topic has a set of linked skill IDs
- a question Suggestion's `target_id` is its corresponding skill ID
- so, you can get all of a topic's questions suggestions by fetching all suggestions with `target_id` in the topic's set of linked skill IDs.

Reply:
Thanks for the explanation! Unfortunately the link doesn't have a reference to Suggestions; but your explanation was quite clear :)